.github/workflows/deploy.yml

```
@@ -4,12 +4,15 @@
-    workflow_dispatch:
+    workflow_run:
+    workflows: ["MyPy", "Pip check", "Unittest", "Unittest Lower Bound"]
+    branches: [ main ]
+    types:
+      - completed
 
 jobs:
-  build:
+  on-success:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
     if: startsWith(github.event.ref, 'refs/tags') || github.event_name == 'release'
```

actionlint (1.7.4)

key "if" is duplicated in "on-success" job. previously defined at line:15,col:5

(syntax-check)